### PR TITLE
Switch over to styled-jsx for themes

### PR DESCRIPTION
**Pull request by captainsafia**

### CODE REVIEW

File: applications/desktop/src/main/menu.js [hunk 1]
```diff
@@ -69,18 +69,6 @@ const theme_menu = [
...
-    label: "Classic",
-    click: createSender("menu:theme", "classic")
-  },
-  {
-    label: "nteract",
-    click: createSender("menu:theme", "nteract")
-  },
-  {
-    label: "One-Dark",
-    click: createSender("menu:theme", "one-dark")
```
Comment (rgbkrk) [inline applications/desktop/src/main/menu.js:83]: Goodbye old themes!

File: applications/desktop/src/main/menu.js [hunk 2]
```diff
@@ -96,20 +84,6 @@ const blink_menu = [
...
-if (month === 12) {
-  theme_menu.push({
-    label: "Hohoho",
-    click: createSender("menu:theme", "christmas")
-  });
-} else if (month === 10) {
-  theme_menu.push({
-    label: "Pumpkin Spice",
-    click: createSender("menu:theme", "halloween")
-  });
```
Comment (rgbkrk) [inline applications/desktop/src/main/menu.js:110]: I'm going to miss Pumpkin Spice most of all.

File: applications/desktop/static/styles/theme-dark.css [hunk 3]
```diff
@@ -1,80 +0,0 @@
...
-
-/*custom scrollbar for notebook cells and hint box*/
-div::-webkit-scrollbar,
-ul::-webkit-scrollbar {
-  width: 11px;
-  height: 11px;
-}
-div::-webkit-scrollbar-track,
-ul::-webkit-scrollbar-track {
-  -webkit-box-shadow: inset 0 0 10px rgba(0, 0, 0, 0.3);
```
Comment (rgbkrk) [inline applications/desktop/static/styles/theme-dark.css:63]: Should we bring these into our styled jsx themes by creating new CSS variables for the scrollbars?

File: applications/desktop/static/styles/theme-light.css [hunk 4]
```diff
@@ -1,86 +0,0 @@
...
-  width: 11px;
-  height: 11px;
-}
-div::-webkit-scrollbar-track,
-ul::-webkit-scrollbar-track {
-  -webkit-box-shadow: inset 0 0 2px rgba(0, 0, 0, 0.3);
-}
-div::-webkit-scrollbar-thumb,
-ul::-webkit-scrollbar-thumb {
-  background-color: rgba(199, 199, 199, 0.4);
```
Comment (rgbkrk) [inline applications/desktop/static/styles/theme-light.css:73]: Yeah I'm thinking we should port these over with new vars.

File: packages/core/src/components/notebook.js [hunk 5]
```diff
@@ -332,11 +334,11 @@ export class Notebook extends React.PureComponent<Props> {
...
           }
         `}</style>
-        <link
-          rel="stylesheet"
-          // TODO: Tear this out or switch to styled-jsx for the inline setting here
-          href={`../static/styles/theme-${this.props.theme}.css`}
-        />
+        <style jsx>{`
+           {
+            this.props.theme: dark;
```
Comment (rgbkrk) [inline packages/core/src/components/notebook.js]: wooooooo, almost there

**Comment by mpacer** [comment]
Made https://github.com/captainsafia/nteract/pull/1 to fix the ternary and imports so that jsx didn't get grumpy. 

The styles still aren't quite… right (e.g., dark doesn't show the text right now) but with 
2622b85 they are loaded correctly.

**Pull request closed without merging**